docs/zh/latest/getting-started.md

```diff
@@ -21,17 +21,29 @@ title: 快速入门指南
 #
 -->
 
-本指南旨在让大家入门 Apache APISIX，我们将配置一个对外提供公共 API 的服务，并由 API key 进行访问保护。
+## 概述
 
-另外，我们将以下面的 `echo` 端点为例，它将返回我们传递的参数。
+本文是 Apache APISIX 的快速入门指南。快速入门分为三个步骤：首先通过[Docker](https://www.docker.com/) 和 [Docker Compose](https://docs.docker.com/compose/) 安装 Apache APISIX，然后创建路由并绑定后端服务，最后使用命令行语句`curl`验证绑定之后返回的结果是否符合预期。
 
-**Request**
+除此之外，本文也提供了 Apache APISIX 的一些进阶操作技巧，包括：添加身份验证、为路由添加前缀、使用 APISIX Dashboard以及常见问题排查。
+
+我们将以下面的 `echo` 端点为例，它将返回我们传递的参数。
+
+**请求内容**
+
+请求 URL 由以下这些参数构成：
+
+- Protocol：即网络传输协议，这里使用的是最常见的HTTP协议
+- Port：即端口，这里使用的`80`端口。
+- Host：即宿主机，这里的主机是`httpbin.org`
+- URI/Path：`/get`
```

Comment:
path? uri?

Reply:
Is it a path or a URI? It is written this way before the change. Please clarify.

Reply:
Fixed, Path.